### ISSUE TITLE
Request for macOS HiDPI support 

### ISSUE THREAD

**Issue by y2-Whyto** [tikzit/tikzit#130]
In the current version, the font size of the code editor down the main graph editor and the descriptions in the style palette are really small on Mac retina display, and I didn’t find any options to configure this in the preferences by now. So could you please add macOS HiDPI support for this application to scale it for retina display? Or consider adding font size configurations in the preferences for all platforms? That will make the drawing experience much more enjoyable. Thank you! 🙏

**Comment by peterkos** [comment]
Agreed -- the UI elements and font size is literally too small for me to use this app.
